src/atoms/toogle/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+
+import * as Styles from './styles';
+import { ToggleProps } from './types';
+
+const Toggle = ({
```

Comment:
```suggestion
const Toggle = ({
  className = 'toggle',
```

Add "toggle" as `className` default.